bech32/bech32_test.go

```diff
@@ -0,0 +1,62 @@
+// Copyright (c) 2017 The btcsuite developers
+// Use of this source code is governed by an ISC
+// license that can be found in the LICENSE file.
+
+package bech32_test
+
+import (
+	"strings"
+	"testing"
+
+	"github.com/btcsuite/btcutil/bech32"
+)
+
+var validBechStrings = []string{
```

Comment:
Please move these definitions inside of the test function scope as to avoid polluting the package namespace.

Ideally, they would be converted to table-driven tests to match almost all the rests of the tests in the code base along the lines of the following, however, I'm fine with the being separate variables so long as they are defined inside the test function scope.

```Go

func TestBech32(t *testing.T) {
	tests := struct{
		str string
		valid bool
	}{
		{"A12UEL5L", true},
		// ...
		{"split1cheo2y9e2w", false},
	}
	for _, test := range tests {
		hrp, decoded, err := bech32.Decode(test.str)
		if test.valid && err != nil {
				t.Errorf("decode of valid string returned unexpected error %v", err)
				continue
		}

		// ....
	}
}